create.rb

```diff
@@ -0,0 +1,112 @@
+class CreatePeople
+  def initialize
+    @books = []
+    @persons = []
+    @rentals = []
+  end
+
+  def list_all_books
+    puts 'Database is empty! Add a book.' if @books.empty?
+    @books.each { |book| puts "[Book] Title: #{book.title}, Author: #{book.author}" }
+  end
+
+  def list_all_persons
+    puts 'Database is empty! Add a person.' if @persons.empty?
+    @persons.each do |person|
+      puts "[#{person.class.name}] Name: #{person.name}, Age: #{person.age}, id: #{person.id}"
+    end
+  end
+
+  def list_all_rentals
+    puts 'To see rentals enter the person ID: '
+    id = gets.chomp.to_i
+
+    puts 'Rented Books: '
+    @rentals.each do |rental|
+      if rental.person.id == id
+        puts "Person: #{rental.person.name} Date: #{rental.date}, Book: '#{rental.book.title}' by #{rental.book.author}"
+      else
+        puts
+        puts 'No record were found for the given ID'
+      end
+    end
+  end
+
+  def create_person
+    print 'press 1 to create a student, press 2 to create a teacher : '
+    option = gets.chomp
+
```

Comment:
- [ ] Great job on the overall refactor. One thing you do need to change to follow the Single responsibility principle is to ensure that all logic relating to requests for input and input validation should've been placed in the main.rb file and/or classes/methods whose responsibilities are to request or validate input. You can refer to [this lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid) to remind yourself about SOLID principles. The purpose of create_person should only be to create the person after it gets relevant data from another function/ class.